### DISCUSSION TITLE
[Feature] Recurse sub-directories for images

### DISCUSSION THREAD

**Discussion by mkjustuk** [immich-app/immich#3485] [Feature Request]
### The feature

For the APP, Google cam puts portrait-mode shots into their own sub-directories within the Camera (DCIM/CAMERA) per shot (pair of images as one has the bokeh and the other doesn't).

Would it be possible to pickup images from not only the Camera (or whatever source is set in the App) directory but then also recurse through any sub-directories for image files, too?

Currently any portrait mode shots are not getting picked up by immich and therefore not backed up.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by sherlock627** [comment]
I concur with this request. My wife has over 200 of these subfolders in her CAMERA folder, and these require manual backup, which is not feasible long term.

**Comment by bo0tzz** [comment]
#1620
